smoke-test/src/test/java/org/opennms/smoketest/OpenNMSCloudPluginIT.java

```diff
@@ -0,0 +1,165 @@
+    private static final String PATH_TO_CLOUD_MOCK = "target/cloud-mock-with-dependencies/org.opennms.plugins.cloud-mock-with-dependencies.jar";
+    private static final String CONTAINER_PATH = "/usr/share/opennms/deploy/";
+    private static final String CLOUD_MOCK_JAR_NAME = "org.opennms.plugins.cloud-mock-with-dependencies.jar";
+    private static final String CLOUD_MOCK_MAIN = "org.opennms.plugins.cloud.ittest.MockCloudMain";
+    private static final String KAR_FILE_NAME = "org.opennms.plugins.cloud-assembly.kar";
+
+    private KarafShell opennmsShell;
+    private KarafShell sentinelShell;
+
+    @ClassRule
+    public static Network network = Network.newNetwork();
+
+    @ClassRule
+    public static OpenNMSStack stack = OpenNMSStack
+            .withModel(StackModel.newBuilder()
+                    .withSentinel()
+                    .build());
+
+    @ClassRule
+    public static GenericContainer mockCloudContainer = new GenericContainer(DockerImageName.parse("opennms/deploy-base:jre-2.0.6.b165"))
+            .withCreateContainerCmdModifier(cmd -> {
+                final CreateContainerCmd createCmd = (CreateContainerCmd) cmd;
+                createCmd.withEntrypoint("/bin/bash");
+                createCmd.withName("cloudMock");
+            }).withCopyFileToContainer(MountableFile.forHostPath(PATH_TO_CLOUD_MOCK), format("%s%s", CONTAINER_PATH, CLOUD_MOCK_JAR_NAME))
+            .withCommand("-c", format("java -cp %s%s %s", CONTAINER_PATH, CLOUD_MOCK_JAR_NAME, CLOUD_MOCK_MAIN)).withExposedPorts(9003)
+            .withNetwork(stack.opennms().getNetwork())
+            .withNetworkAliases("cloudMock");
+
+    @Before
+    public void setUp() {
+        this.opennmsShell = new KarafShell(stack.opennms().getSshAddress());
+        this.sentinelShell = new KarafShell(stack.sentinel().getSshAddress());
+        mockCloudContainer.start();
+    }
+
+    @Test
+    public void installCloudPlugin() throws Exception {
+        install_cloud_plugin_in_opennms_must_be_successful();
+        install_cloud_plugin_in_sentinel_must_be_successful();
+    }
+
+    public void install_cloud_plugin_in_opennms_must_be_successful() throws Exception {
+        // Given
+        stack.opennms().copyFileToContainer(MountableFile.forHostPath(PATH_TO_FILE), format("%s%s", CONTAINER_PATH, KAR_FILE_NAME));
+        String config = createConfig();
+
+        // When
+        opennmsShell.runCommand(String.format("kar:install file:%s%s", CONTAINER_PATH, KAR_FILE_NAME));
+        opennmsShell.runCommand("feature:install opennms-plugin-cloud-core");
+        opennmsShell.runCommand("feature:list | grep opennms-plugin-cloud-core", output -> output.contains("Started"));
+        opennmsShell.runCommand(config);
+        opennmsShell.runCommand("opennms-cloud:init key", output -> output.contains("Initialization of cloud plugin in OPENNMS was successful."));
+    }
+
+
+    public void install_cloud_plugin_in_sentinel_must_be_successful() throws Exception {
```

Comment:
Please stick to java standard method name scheme.

Reply:
ok